guides/v2.2/cloud/cdn/configure-fastly.md

```diff
@@ -23,185 +23,113 @@ configurations.
+
+  - `CONFIG__DEFAULT__SYSTEM__FULL_PAGE_CACHE__FASTLY__FASTLY_API_KEY`
+
+  - `CONFIG__DEFAULT__SYSTEM__FULL_PAGE_CACHE__FASTLY__FASTLY_SERVICE_ID`
 
 {:.bs-callout .bs-callout-info}
-If you need to change the Fastly API key credential for a project environment for security reasons, you can request a new API key from support.  See [Change Fastly credentials](https://support.magento.com/hc/en-us/articles/360006935271-Change-account-owner-access-credentials-via-API-tokens-for-Fastly-on-Cloud).
+If you cannot find the Fastly credentials for the Staging or Production environments, contact your Magento Technical Account Manager.
 
-## Get started {#cloud-fastly-start}
+## Enable Fastly caching for your Cloud environments {#cloud-fastly-config}
 
-You need to install Fastly in its own branch. Fine-tuning Fastly can be a
-complex process, depending on your needs and store size. If you already have a
-branch to work in, or know how to create a branch, continue to
-[installing Fastly](#cloud-fastly-setup).
+**Prerequisites**
 
-To create a branch:
+- [Fastly CDN for Magento 2 module]({{ page.baseurl }}/cloud/cdn/cloud-fastly.html#fastly-cdn-module-for-magento-2) installed in the Staging and Production environment
 
-{% include cloud/cli-get-started.md %}
+- [Fastly credentials](#cloud-fastly-creds) for {{ site.data.var.ece }} Staging and Production environments
 
-## Install Fastly in an Integration branch and deploy {#cloud-fastly-setup}
+#### To enable Fastly CDN caching in Staging and Production
 
-You should install the Fastly module on your local, pushing the code to
-Integration and deploying across to your Staging and Production environments.
-For {{site.data.var.ece}} 2.2, install Fastly module 1.2.33 or later for all
-updated settings and full VCL snippet uploading support.
+{% include cloud/admin-ui-login-step.md %}
 
-{:.bs-callout .bs-callout-warning}
-Don't configure the module in your local before building and deploying. You'll
-configure the module in those environments.
+1. Click **Stores** > Settings > **Configuration** > **Advanced** > **System** and expand **Full Page Cache**.
 
-We recommend using the [`bin/magento magento-cloud:scd-dump`]({{ page.baseurl }}/cloud/live/sens-data-over.html#cloud-config-specific-recomm) command for
-Configuration Management. When you use the `app:config:dump` command, all configuration options for Fastly
-are locked and cannot be edited in Staging and Production.
+   ![Expand to select Fastly]({{ site.baseurl }}/common/images/cloud_fastly_menu.png){:width="650px"}
 
-We provide Fastly services only for your Staging and Production environments.
-You cannot use the Fastly service in Integration environments.
+1. In the **Caching Application** section, remove the selection from **Use system value**, and then select **Fastly CDN** from the drop-down list.
```

Comment:
In a step, a section seems like a location instead of something that requires action.

```suggestion
1. In the _Caching Application_ section, remove the selection from **Use system value**, and then select **Fastly CDN** from the drop-down list.
```